azure-pipelines-product-construction-service.yml

```diff
@@ -31,6 +31,8 @@ variables:
     value: productconstructionint.azurecr.io
   - name: serviceConnectionName
     value: ProductConstructionServiceDeploymentInt
+  - name: authServiceConnection
+    value: "Darc: Maestro Staging"
```

Comment:
We only use staging and never interact with the prod app?

Reply:
for now, yes